src/Microsoft.OData.Edm/Validation/EdmErrorCode.cs

```diff
@@ -579,10 +579,10 @@ public enum EdmErrorCode
         /// </summary>
         InvalidPolymorphicComplexType = 221,
 
-        /// <summary>
-        /// A navigation property without direct containment cannot contain its declaring entity indirectly.
-        /// </summary>
-        NavigationPropertyEntityMustNotIndirectlyContainItself = 222,
+        // /// <summary>

```

Comment:
Is this commented, rather than removed, in order to make sure we don't re-use the value 222? Should we add a comment to that effect?

Reply:
Yes. removing it is a breaking change. I will keep it unchanged.